fga-eps-mds/2020.1-stay-safe-user-service
# Favorite Places Endpoint

## Issue 
Resolves #39 


## Description

Implemented create, read and delete favorite user locations. 
The endpoints are protected with authentication and it is not possible for a user to change another user's favorite locations.
Updated swagger documentation accordingly.

## Checklist:



## How to validate?

Check in route 'api/places/' if get, post and delete methods are working correctly.
Run the tests.
